content/blog/mlops-–-deploying-an-ml-model-in-greenlake-platform-mlops-service.md

```diff
@@ -0,0 +1,318 @@
+
+        name: tritonservercont
+
+        command: ["/bin/bash"]
+
+        args: ["-c", "/opt/tritonserver/bin/tritonserver --model-repository=s3://https://<objectstoreurl.com:port>/sample/models --strict-model-config=false"]
+
+        env:
+
+        - name: AWS_ACCESS_KEY_ID
+
+            valueFrom:
+
+              secretKeyRef:
+
+                name: minio-cred
+
+                key: AWS_ACCESS_KEY_ID
+
+        - name : AWS_SECRET_ACCESS_KEY
+
+            valueFrom:
+
+              secretKeyRef:
+
+                name: minio-cred
+
+                key: AWS_SECRET_ACCESS_KEY
+
+        ports:
+
+        - containerPort: 8000
+
+            name: http
+
+        - containerPort: 8001
+
+            name: grpc
+
+        - containerPort: 8002
+
+            name: metrics
+
+        volumeMounts:
+
+        - mountPath: /dev/shm
+
+          name: dshm
+
+        resources:
+
+          requests:
+
+            cpu: 2
+
+            memory: 8Gi
+
+            nvidia.com/gpu: 1
+
+          limits:
+
+            cpu: 2
+
+            memory: 16Gi
+
+            nvidia.com/gpu: 1
+
+      volumes:
+
+      - name: dshm
+
+        emptyDir:
+
+          medium: Memory
+
+      securityContext:
+
+        runAsUser: 1000
+
+        fsGroup: 1000
+
+      nodeSelector:
+
+        gl.hpe.com/instance-type: GL-GP-MLi-Metal
+
+---
+
+apiVersion: v1
+
+kind: Service
+
+metadata:
+
+  name: triton
+
+  namespace: triton
+
+  labels:
+
+    hpecp.hpe.com/hpecp-internal-gateway: "true"
+
+spec:
+
+  type: NodePort
+
+  selector:
+
+    app: triton
+
+  ports:
+
+  - protocol: TCP
+
+      name: http
+
+      port: 8000
+
+      nodePort: 30850
+
+      targetPort: 8000
+
+  - protocol: TCP
+
+      name: grpc
+
+      port: 8001
+
+      nodePort: 30851
+
+      targetPort: 8001
+
+  - protocol: TCP
+
+      name: metrics
+
+      nodePort: 30852
+
+      port: 8002
+
+      targetPort: 8002
+```
+
+          **Notes:**
+
+* Replace <objectstoreurl.com:port> and model path with actual object store url, model sub path where model and associate configuration is placed
+* Node selector “gl.hpe.com/instance-type: GL-GP-MLi-Metal” is used to place the workload in inference cluster
+* Check the pods and service are running by running the below commands:
+
+```shell
+kubectl get po –n triton
+
+kubectl get svc –n triton
+```
+
+* Service labels **hpecp.hpe.com/hpecp-internal-gateway: "true"** is placed to get a gateway endpoint to access the triton inference server outside the cluster. To find the endpoint run command 
+
+```shell
+kubectl describe svc <service_name> -n triton
+```
+
+* Check the metrics endpoints for models hosted in triton inference server are accessible:
+
+```shell
+kubectl describe svc <service_name> -n triton
+```
+
+**Next steps**
+
+Once the production model is hosted, then any application can perform inference on the model hosted by Triton inference server. More on Triton client libraries refer the references section below
```

Comment:
Inference?
Second sentence, recommend rephrasing to "For more on Triton client libraries, refer to the section below." (add the word More, a comma and shorten sentence.)